sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/ReceiptExtensions.java

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.FormField;
+import com.azure.ai.formrecognizer.models.RecognizedReceipt;
+import com.azure.ai.formrecognizer.models.USReceipt;
+import com.azure.ai.formrecognizer.models.USReceiptItem;
+import com.azure.ai.formrecognizer.models.USReceiptType;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static com.azure.ai.formrecognizer.models.ReceiptItemType.NAME;
+import static com.azure.ai.formrecognizer.models.ReceiptItemType.PRICE;
+import static com.azure.ai.formrecognizer.models.ReceiptItemType.QUANTITY;
+import static com.azure.ai.formrecognizer.models.ReceiptItemType.TOTAL_PRICE;
+
+/**
+ * The ReceiptExtensions class to allow users to convert a {@link RecognizedReceipt}
+ * to a receiptLocale specific Receipt type.
+ */
+@SuppressWarnings("unchecked")
+public final class ReceiptExtensions {
+
+    /**
+     * Static method to convert an incoming receipt to a {@link USReceipt type}.
+     *
+     * @param receipt The {@link RecognizedReceipt recognized receipt}.
+     *
+     * @return The converted {@link USReceipt US locale receipt} type.
+     */
+    public static USReceipt asUSReceipt(RecognizedReceipt receipt) {
```

Comment:
As per naming conventions, acronyms should follow camel-case convention but in this case, I am not sure naming this as `asUsReceipt` is good.

Reply:
I think that is what we had decided as a team. Do you have any suggestions we discuss it with the crew.

Reply:
Either leave `US` as-is or expand it. Don't really like changing this to `Us`. Check with @JonathanGiles  too.